accumulator/transformreadme.md

```diff
@@ -0,0 +1,100 @@
+Transform is used to calculate where each leaves should move to during a
+batch deletion. This significantly increases performance during deletions
+compared to deleting leaves one by one. No actual deletions happen in a tree
+during this phase.
+
+RemTrans() is the main workhorse for the transform step.
+
+
+The deletions are marked from the bottom to the top row. We go through row
+with the steps outlined below.
+
+1) DelRoot:
+The roots are deleted before progressing. This is because they will be incorrect
+as the lower hashes will change. 
+
+Here we have a tree of height 2. When RemTrans is on row 2, 06 will be
```

Comment:
it would be clearer if the row numbers are annotated in the diagrams. so that people know if it decreasing or 0/1-indexed

Reply:
Addressed